test/test_other.py

```diff
@@ -8098,12 +8096,24 @@ def test_wasm_exceptions_stack_trace_and_message(self):
     #     at doRun (test.js:4621:23)
     #     at run (test.js:4636:5)
     stack_trace_checks = [
-      'std::runtime_error,my message',
-      'at __cxa_throw',
+      'std::runtime_error[:,][ ]?my message',  # 'std::runtime_error: my message' for Emscripten EH
```

Comment:
```suggestion
      'std::runtime_error[:,][ ]?my message',  # 'std::runtime_error: my message' for Wasm EH
```
Isn't this Wasm EH?

Reply:
No, this is specific to Emscripten EH. For example Emscripten EH would give this stacktrace and message:
```
std::runtime_error: my message
     at ___cxa_throw (test.js:1526:13)
...
```

Whereas Wasm EH would produce:
```
exiting due to exception: [object WebAssembly.Exception],Error: std::runtime_error,my message
     at __cxa_throw (wasm://wasm/009a7c9a:wasm-function[1551]:0x24367)
...
```

Reply:
I see, thanks.